src/components/input/DurationInput.js

```diff
@@ -140,6 +149,7 @@ const DurationInput = ({ value, onChange, onBlur, onFocus, onKeyDown, autoFocus,
         placeholder="00"
         step={MINUTES_STEP}
         {...(!value?.hours ? { min: 0 } : {})}
+        {...(maxHours && value?.hours && value.hours >= maxHours ? { max: maxMinutes } : {})}
```

Comment:
Not sure I'm following with this one. Doesn't this mean that if you have a max with less than 59 minutes (thus being able to not fill in the hours), the max will never matter?

Reply:
Oh, you're right. Will fix!